### PR TITLE
use existing files in dir instead of tmp file if here() is non-empty

### PR DESCRIPTION
**Pull request by AnjaMinsaas**

### CODE REVIEW

File: R/here.R [hunk 1]
```diff
@@ -31,9 +31,15 @@ with_here <- function(new_here, expr, chdir=FALSE, verbose=FALSE ) {
 
     current_here <- here()
 
-    # create a temporary file to get us back when we're done
-    tf_current <- local_tempfile(tmpdir=current_here, pattern=".here")
-    file_touch(tf_current)
+    # find or make a suitable file to get us back when we're done
+    if(length(dir(current_here)) >= 1){
```
Comment (torbjorn) [inline R/here.R]: style, as discussed

Comment (AnjaMinsaas) [reply R/here.R]: done

File: R/here.R [hunk 2]
```diff
@@ -31,9 +31,15 @@ with_here <- function(new_here, expr, chdir=FALSE, verbose=FALSE ) {
 
     current_here <- here()
 
-    # create a temporary file to get us back when we're done
-    tf_current <- local_tempfile(tmpdir=current_here, pattern=".here")
-    file_touch(tf_current)
+    # find or make a suitable file to get us back when we're done
+    if(length(dir(current_here)) >= 1) {
+        # fish out the first file in here()
+        tf_current <- dir(current_here)[1]
```
Comment (torbjorn) [inline R/here.R]: we've alrady imported fs, might as well use `dir_ls()` , also based on experience, it seems `i_am()` needs a file, not a directory, so dir_ls() output needs to be filtered for files. (eg. file_exists)

Comment (torbjorn) [reply R/here.R]: .. and use all=TRUE , a dot file is as good as any file

**Review by torbjorn — APPROVED** [review]
Looks good!

**Pull request merged**